src/ActivityRepository.js

```diff
@@ -62,40 +62,22 @@ class ActivityRepository {
     };
   }
 
-  getAverageStairsDay(date) {
+  getAverages(date, property) {
     const filteredDate = this.getFilteredDate(date);
-    const totalStairs = filteredDate.reduce((total, log) => {
-      total += log.flightsOfStairs;
+    const total = filteredDate.reduce((total, log) => {
```

Comment:
Great job with this, that cut out a lot.

Reply:
🎉